readme.md

```diff
@@ -53,7 +54,10 @@ body{
 }
 ```
```

Comment:
Re Question 3, The snippet 
```
var theBody = document.querySelectorAll("body");
theBody.style.backgroundColor = "red";
```
throws an error because of an unexpected method `style` for the element you get back from `document.QuerySelectorAll("body")`
```
This is because `querySelectorAll` always returns an array of elements, even when that array includes just one element. This can be fixed with `document.body`, `document.querySelectorall("body")[0]`, or `document.querySelector("body")`